dockerfiles/mlrun-api/requirements.txt

```diff
@@ -12,4 +12,4 @@ sqlalchemy~=1.4
 pymysql~=1.0
 alembic~=1.9
 timelength~=1.1
-memray~=1.12
+memray~=1.12; sys_platform != 'win32'
```

Comment:
Why is `memray` in the requirements? Shouldn’t it be in the dev-requirements?

Reply:
it is needed on the API image to monitor the API memory